Use versioned IRIs in rdfs:isDefinedBy (#1169)
semanticarts/gist: docs/release_notes/issue383-versioned-defined-by.md

### Patch Updates

> - Changed the values of `rdfs:isDefinedBy` from the unversioned to the versioned gist IRI; i.e., in this release, to `https://w3id.org/semanticarts/ontology/gistCore13.1.0`. Issue [#383](https://github.com/semanticarts/gist/issues/383).

Reviewer: I think we are going to version 14.0.0 next?
```suggestion
- Changed the values of `rdfs:isDefinedBy` from the unversioned to the versioned gist IRI; i.e., to `https://w3id.org/semanticarts/ontology/gistCore13.1.0`. Issue [#383](https://github.com/semanticarts/gist/issues/383).
```

> - Changed the values of `rdfs:isDefinedBy` from the unversioned to the versioned gist IRI: that is, in this release, to `https://w3id.org/semanticarts/ontology/gistCore13.1.0`. Issue [#383](https://github.com/semanticarts/gist/issues/383).

Author: ```suggestion
- Changed the values of `rdfs:isDefinedBy` from the unversioned to the versioned gist IRI; i.e., in this release, to `https://w3id.org/semanticarts/ontology/gistCore13.1.0`. Issue [#383](https://github.com/semanticarts/gist/issues/383).
```
